docs/specs/user-profile/design.md

```diff
@@ -0,0 +1,331 @@
+# User profile
+
+User profile consists of standard attributes and custom attributes.
+
+## Standard attributes
+
+Standard attributes are [OIDC standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).
+All standard attributes are optional.
+
+## Access control on standard attributes
+
+The Admin API and the portal always have full access to all standard attributes.
+
+```yaml
+user_profile:
+  standard_attributes:
+    access_control:
+    - pointer: /given_name
+      access_control: hidden
+    - pointer: /zoneinfo
+      access_control: readonly
+```
+
+Possible values of `access_control` are:
+
+- `hidden`: The standard attribute is hidden from the end-user, the User Info endpoint and the resolver.
+- `internal`: The standard attribute is hidden in the settings page. But it is visible to the User Info endpoint and the resolver.
+- `readonly`: The standard attribute is visible to the end-user, the User Info endpoint and the resolver. The end-user can view but cannot edit it in the settings page.
+- `readwrite`: The standard attribute is visible to the end-user, the User Info endpoint and the resolver. The end-user can view and edit it in the settings page.
+
+The default value of `access_control` of all standard attributes is `readwrite`.
+
```

Comment:
Recap discussion in standup:
May need a config to control visibility in the Admin Portal too. 
As we provide input fields for changing these values. When using the portal for user management, the staff will feel overwhelmed by the attributes which are not used by their app at all.